test/adapters/stdio/read.stdio.spec.js

```diff
@@ -5,7 +5,7 @@ var juttle_test_utils = require('../../runtime/specs/juttle-test-utils');
 var check_juttle = juttle_test_utils.check_juttle;
 var path = require('path');
 
-var validFormats = {
+var symmetricalFormats = {
```

Comment:
This naming change seems superfluous but more importantly it seems you should be consistent and change the file test if you're gonna change this one.